frontend/src/services/loginServices.js

```
@@ -84,4 +84,13 @@ export const getCurrentUser = async () => {
   }
 };
 
+export const getHasUsers = async () => {
+  try {
+    const response = await apiClient.get("users/has-users");
+    return response.data;
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)